### ISSUE TITLE
Any eBook Converter Using this Library/Repo

### ISSUE THREAD

**Issue by asheroto** [apprenticeharper/DeDRM_tools#1993]
Not sure if you knew this, or if the license permits, but this software is using this library of tools and selling the tools for purchase....

https://www.any-ebook-converter.com

![image](https://user-images.githubusercontent.com/49938263/149806262-969cfa10-a74a-44b7-b108-e99a83cf93f9.png)

**Comment by ElleKayEm** [comment]
Yes, the commercial DRM removers are more than likely all based on this code.  Since DRM removal is a legal grey area in many places, not much to be done.  Buyer beware.
